pages.fr/common/git-pull.md

```diff
@@ -0,0 +1,16 @@
+# git pull
+
+> Récupère une branche depuis le serveur distant et fusionne  la dans le serveur local
```

Comment:
```suggestion
> Récupère une branche depuis le serveur distant et fusionne  la dans le serveur local.
```
Missing ending period.

Reply:
There's no "local server" here, it's more of a "local branch" instead.